gazelle/update_repos.go

```diff
@@ -272,5 +272,5 @@ const bzlmodUseRepoNamesUpdMarkerEnd = "    # swift_deps bzlmod use_repo END\n"
 const bzlmodStanzasUpdMarkerStart = "# swift_deps START\n"
 const bzlmodStanzasUpdMarkerEnd = "# swift_deps END\n"
 
-const bzlmodInstructions = `If you have enabled bzlmod, add the following to your 'MODULE.bazel' file to 
+const bzlmodInstructions = `If you have enabled bzlmod, add the following to your 'MODULE.bazel' file to
```

Comment:
Was the removal of this space intentional? I think that it will concatenate `toload`.

Reply:
Hmm nope looks like either me or my editor did that, will revert.